website/docs/quick-start-guide.md

```diff
@@ -1155,7 +1163,13 @@ delete from hudi_cow_pt_tbl where name = 'a1';
 
 </TabItem>
 <TabItem value="python">
-Delete records for the HoodieKeys passed in.<br/>
+
+#### Code overview 
```

Comment:
This seems like redundant info already available in the lines 962 - 965 above. Do we need this specifically for Python tab?

Reply:
This is just like a friendly reminder about the code below, so user doesn't have to scroll up and refresh